test/framework/easyconfig.py

```diff
@@ -839,13 +857,6 @@ def test_obtain_easyconfig(self):
         self.assertEqual(ec['start_dir'], specs['start_dir'])
         remove_file(res[1])
 
-        specs.update({
-            'foo': 'bar123'
-        })
-        self.assertErrorRegex(EasyBuildError, "Unknown easyconfig parameter: foo",
```

Comment:
I would think this test is still useful for parameters that are not defined in the easyblock, or is that taken care of somewhere else?

Reply:
That is taken care of elsewhere. I don't think we can trigger this error anymore at all, as that is specific to using non-default parameters which we allow now.
The EasyBlock instantiation will trigger e.g. "Use of 2 unknown easyconfig parameters detected in test.eb: foo, some_list", see test_fix_deprecated_easyconfigs